api/v1alpha1/workspace_condition_types.go

```diff
@@ -16,6 +16,9 @@ const (
 	// WorkspaceConditionTypeInferenceStatus is the state when Inference has been created.
 	WorkspaceConditionTypeInferenceStatus = ConditionType("InferenceReady")
 
+	// WorkspaceConditionTypeTuningStatus is the state when Tuning has been created.
+	WorkspaceConditionTypeTuningStatus = ConditionType("TuningReady")
```

Comment:
For job type workload, the status cannot be "READY".  The status can be "STARTED" and "DONE". Check what status code the build-in k8s job uses.

Reply:
Done